client/src/pages/course/admin/tasks.tsx

```diff
@@ -145,23 +156,30 @@ function Page(props: CoursePageProps) {
 
   const handleCrossCheckDistribution = async (record: CourseTaskDetails) => {
     try {
-      await service.createCrossCheckDistribution(record.id);
-      message.success('Cross-Check distrubtion has been created');
+      const {
+        data: { crossCheckPairs },
+      } = await service.createCrossCheckDistribution(record.id);
+      if (crossCheckPairs.length) {
+        message.success('Cross-Check distrubtion has been created');
+      } else {
+        message.warn('Unable to create Cross-Check distribution: no solutions submitted');
+      }
     } catch (e) {
       message.error('An error occurred.');
+    } finally {
+      await loadData();
```

Comment:
same comment as below

Reply:
loadData uses spinner by itself, and we are only interested in geting fresh data after request ends somehow, there's no need for 2 consecutive spinners
![image](https://user-images.githubusercontent.com/34455330/178045302-275a5a0b-b307-4e8b-b66e-afe82894205d.png)